lib/entities/workflow-definition.ts

```diff
@@ -13,6 +13,34 @@ import { wrapCollection } from '../common-utils'
 import enhanceWithMethods from '../enhance-with-methods'
 import { ActionType } from './role'
 
+/* Workflow Step */
+
+export type WorkflowStepAction = {
+  action: Link<'AppAction'>
+  body: string
+  headers: Record<string, unknown>
```

Comment:
I think headers can be of type `Record<string, string>`

Reply:
I'm not sure if this is the same purpose but we use this headers type for every `headers` property in CMA: https://github.com/contentful/contentful-management.js/blob/724ff87a48de1d55d21392af198ed5e58be008d9/lib/common-types.ts#L578